doc/api/errors.md

```diff
@@ -716,14 +766,32 @@ synchronous forked Node.js process. See the documentation for the
 <a id="ERR_MISSING_ARGS"></a>
 ### ERR_MISSING_ARGS
 
-Used when a required argument of a Node.js API is not passed. This is currently
-only used in the [WHATWG URL API][] for strict compliance with the specification
-(which in some cases may accept `func(undefined)` but not `func()`). In most
-native Node.js APIs, `func(undefined)` and `func()` are treated identically, and
-the [`ERR_INVALID_ARG_TYPE`][] error code may be used instead.
+Used when a required argument of a Node.js API is not passed. This is only used
+for strict compliance with the API specification (which in some cases may accept
+`func(undefined)` but not `func()`). In most native Node.js APIs,
+`func(undefined)` and `func()` are treated identically, and the
+[`ERR_INVALID_ARG_TYPE`][] error code may be used instead.
+
+<a id="ERR_MULTIPLE_CALLBACK"></a>
+### ERR_MULTIPLE_CALLBACK
+
+Used when a callback is called more than once. *Note*: A callback is meant to
```

Comment:
Please move the `*Note*:` to it's own paragraph... e.g.

```
Used when a callback is called more than once.

*Note*: A callback ...
```

Also, *some* (albeit a very small number) callbacks may be called multiple times but generally should only be called once. Perhaps augment this a bit by saying `Most callbacks are meant to be called only once...`

Also `s/more then/more than`